spec/flipper/feature_spec.rb

```diff
@@ -270,9 +270,9 @@
         @preview_features = Flipper.register(:preview_features) { |thing| true }
         @not_enabled = Flipper.register(:not_enabled) { |thing| true }
         @disabled = Flipper.register(:disabled) { |thing| true }
-        subject.enable @staff
-        subject.enable @preview_features
-        subject.disable @disabled
+        subject.enable Flipper::Types::Group.new(:staff)
```

Comment:
https://github.com/aroben/flipper/commit/95ca7f43af4327dffb99b20aa0f001010c67f40e made `Flipper.register` return a `Flipper::Group` instead of a `Flipper::Types::Group`. This code was then passing that `Flipper::Group` into `Feature#enable`.

We could handle this case, but it seemed like it would just encourage mistakes to be so lax about the types. I think in most cases client code shouldn't be dealing with `Flipper::Group` objects at all; they're mostly an implementation detail.